helpers.go

```diff
@@ -60,6 +61,46 @@ func isIgnoredPod(name string) bool {
 	return false
 }
 
+func isIgnoredErrorForPod(podName string, errorLog string) bool {
+	ignoredErrorsForPodNamePrefixesEnv := os.Getenv("IGNORED_ERRORS_FOR_POD_NAME_PREFIXES")
+	if ignoredErrorsForPodNamePrefixesEnv == "" {
+		return false
+	}
+
+	podErrorsMap := make(map[string][]interface{})
+	err := json.Unmarshal([]byte(ignoredErrorsForPodNamePrefixesEnv), &podErrorsMap)
+	if err != nil {
+		klog.Infof("Failed to load IGNORED_ERRORS_FOR_POD_NAME_PREFIXES with error: %s", err)
+		return false
+	}
+
+	for key, errors := range podErrorsMap {
+		if strings.HasPrefix(podName, key) {
+			for _, ignoredError := range errors {
+				if strings.Contains(errorLog, ignoredError.(string)) {
+					klog.Infof("Ignore: pod %s has ignored error: %s\n", podName, ignoredError)
+					return true
+				}
+			}
+		}
+	}
+
+	return false
+}
+
+func lastNonEmptyLogLine(logs string) string {
```

Comment:
We check only the last non-empty log line?
It is a little bit hard to specify the error message in the last line.
How about checking all the line in containerLogs?